hardware/raspberrypi/booteeprom.md

```diff
@@ -21,14 +21,15 @@ To check that the bootloader is working correctly, turn off the power, unplug ev
 
 If the Raspberry Pi is not booting it's possible that the bootloader EEPROM is corrupted. This can easily be reprogrammed using the Raspberry Pi Imager tool which is available via the [raspberrypi.org downloads page](https://www.raspberrypi.org/downloads/).
 
+Using the recovery image will erase any custom configuration options, resetting the bootloader back to factory defaults.
+
 ## Updating the bootloader
 
-We recommend setting up your Pi so that it automatically updates the bootloader: this means you will get new features and bug fixes as they are released. Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates.
+We recommend setting up your Pi so that it automatically updates the bootloader: this means you will get new features and bug fixes as they are released. Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates. `rpi-eeprom` is available in the latest Raspberry Pi OS releases.
```

Comment:
"We recommend setting up your Pi so that it automatically updates the bootloader"

If `rpi-eeprom` is now installed by default, I guess this happens automatically, so there's no "setting up" to do?

Reply:
Yes, I'd forgotten that all of this stuff was added post-launch. Let's just baseline the docs at the latest Pi OS release / stable update where rpi-eeprom is already installed and network boot and HDMI diagnostics are standard.